core/deployment/src/main/java/io/quarkus/deployment/console/ConsoleStateManager.java

```diff
@@ -70,7 +70,13 @@ public void accept(int[] ints) {
                     Holder command = commands.get(key);
                     if (command != null) {
                         if (command.consoleCommand.getReadLineHandler() != null) {
-                            QuarkusConsole.INSTANCE.doReadLine();
+                            if (key == 'e') {
+                                QuarkusConsole.INSTANCE.doReadLineWithPrompt("Current args: "
```

Comment:
what do you think about just having it be:

`"state list"> `

instead of:

`Current args: "state list">` 

just to save some characters in width?

otherwise maybe split over two lines and not use `current` as you can't "escape" to reuse it?

`Enter new arguments (was "state list"):
>`

Reply:
Prompt changed